locales/templates/app.pot

```diff
@@ -645,6 +657,16 @@ msgctxt "screenshot_image_viewer_dialog_delete_msg"
 msgid "Delete this image?"
 msgstr ""
 
+msgctxt "screenshot_on_boarding_text_msg"
+msgid ""
+"Tap the screenshot button in the menu to capture the whole page.\n"
+"Fast, easy, and come back to the page at anytime!"
```

Comment:
This sentence sounds really weird. The fact that there are 2 nouns and then a verb is very surprising. It should rather be something like "Fast and easy. Come back to the page at anytime!" or maybe "Fast, easy - come back to the page at anytime!"

Reply:
How about let's go with this one?
Fast, easy - come back to the page at anytime!